### PR TITLE
English Changelog and links

### PR DESCRIPTION
**Pull request by timse201**
Whole Changelog now in english

Link to the Github Repository and WordPress Translate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,9 +36,10 @@ Smart, efficient cache solution for WordPress. Use DB, HDD, APC or Memcached for
 * We don’t handle support via e-mail, Twitter, GitHub issues etc.
 
 ### Contribute ###
-* Active development of this plugin is handled on GitHub.
+* Active development of this plugin is handled on GitHub. (see here: https://github.com/pluginkollektiv/cachify)
```
Comment (swissspidy) [inline README.md]: As this is Markdown, I'd rather use links à la `[…] handled [on GitHub](https://github.com/pluginkollektiv/cachify).`

File: README.md [hunk 2]
```diff
@@ -36,9 +36,10 @@ Smart, efficient cache solution for WordPress. Use DB, HDD, APC or Memcached for
...
 * Pull requests for documented bugs are highly appreciated.
 * If you think you’ve found a bug (e.g. you’re experiencing unexpected behavior), please post at the [support forums](https://wordpress.org/support/plugin/cachify) first.
+* If you want to help us translate this plugin you can do so here: https://translate.wordpress.org/projects/wp-plugins/cachify
```
Comment (swissspidy) [inline README.md]: Same here. Maybe something along the lines of `[…] translate this plugin, head over to [translate.wordpress.org](https://translate.wordpress.org/projects/wp-plugins/cachify).`

File: README.md [hunk 3]
```diff
@@ -69,15 +70,15 @@ Smart, efficient cache solution for WordPress. Use DB, HDD, APC or Memcached for
...
 
 ### 2.2.2 ###
-* Fix: Parameter-Rückgabe beim Filter `dashboard_glance_items`
-* Großzügige Anwendung des Filters `esc_html`
+* Fix: parameter response by filter `dashboard_glance_items`
+* Generous use of the filter `esc_html`
 
 ### 2.2.1 ###
-* Fix für die Meldung "Call to undefined function is_plugin_active_for_network" in WordPress-Multisite
+* Fix for the note "Call to undefined function is_plugin_active_for_network" on WordPress-Multisites
```
Comment (swissspidy) [inline README.md]: I assume it was originally a PHP _notice_, not _note_.

Also: _WordPress Multisite_, not _WordPress-Multisites_

File: README.md [hunk 4]
```diff
@@ -69,15 +70,15 @@ Smart, efficient cache solution for WordPress. Use DB, HDD, APC or Memcached for
 * Updated [plugin authors](https://gist.github.com/glueckpress/f058c0ab973d45a72720)
 
 ### 2.2.2 ###
-* Fix: Parameter-Rückgabe beim Filter `dashboard_glance_items`
-* Großzügige Anwendung des Filters `esc_html`
+* Fix: parameter response by filter `dashboard_glance_items`
```
Comment (swissspidy) [inline README.md]: It's about _returning_ a parameter in that filter, not a _response_.

**Comment by swissspidy** [comment]
Thank you very much for your pull request! I added some notes directly inline. It would be great if you could adjust the wording there in a new commit.

**Comment by timse201** [comment]
ok should we drop the changelog.md and insert the missing entrys?

**Comment by swissspidy** [comment]
I created `CHANGELOG.md` on purpose, because otherwise people have to translate dozens of lines of changelog entries which aren't that important. A very cumbersome process. The most recent changelog entries are to only ones people (should) care about.

File: README.md [hunk 5]
```diff
@@ -74,7 +74,7 @@ Smart, efficient cache solution for WordPress. Use DB, HDD, APC or Memcached for
 * Generous use of the filter `esc_html`
 
 ### 2.2.1 ###
-* Fix for the PHP notice "Call to undefined function is_plugin_active_for_network" on WordPress-Multisites
+* Fix for the PHP notice "Call to undefined function is_plugin_active_for_network" on WordPress-Multisite
```
Comment (swissspidy) [inline README.md]: Can you remove the dash too? Sorry for nitpicking again 🙂

**Comment by timse201** [comment]
no problem
should i insert english translation there?

**Comment by swissspidy** [comment]
Not sure… I think we can keep it for now as it's not really worth the effort. Think of it more like an archive.

**Comment by timse201** [comment]
ok

**Pull request merged**